src/Microsoft.Health.Dicom.Core/Modules/MediationModule.cs

```diff
@@ -23,7 +23,7 @@ public void Load(IServiceCollection services)
 
         Assembly coreAssembly = typeof(MediationModule).Assembly;
 
-        services.AddMediatR(GetType().Assembly, coreAssembly);
```

Comment:
You also need to invoke `RegisterServicesFromAssemblyContaining(GetType())` in the `AddMediatR` delegate

Reply:
Wait... aren't these the same assemblies...? lol

Reply:
We can replace with just `RegisterServicesFromAssemblyContaining`

Reply:
Changed